app/concepts/user/create.rb

```diff
@@ -0,0 +1,19 @@
+class User::Create < Trailblazer::Operation
+  include Model
+  model User, :create
+
+  contract do 
+    property :fullname
+    validates :fullname, presence: true
+  end
+  
+  def process(params)
+    params[:todo][:fullname] = params[:todo][:fullname].blank? ? "Guest" : params[:todo][:fullname]
```

Comment:
nice! The idea is already here. Just a few params setup on board, and this would be good. Initially, you can maximise Trailblazer operation when handling, manipulating and specifying `params` values.